migrate.py

```diff
@@ -658,10 +662,10 @@ def format_user(user, options):
     bb_user = "Bitbucket: [{0}](https://bitbucket.org/{0})".format(user['nickname'])
     gh_username = _gh_username(user['nickname'], options.users, options.gh_auth)
     if gh_username is not None:
-        gh_user = "GitHub: [{0}](https://github.com/{0})".format(gh_username)
+        gh_user = ", GitHub: [{0}](https://github.com/{0})".format(gh_username)
     else:
         gh_user = ""
-    return (user['display_name'] + " (" + bb_user + ", " + gh_user + ")")
+    return "{0} ({1}{2})".format(user['display_name'], bb_user, gh_user)
```

Comment:
Can we use the nifty new `f` strings on this one? To make it more readable?

Reply:
Happy to; let me know your thoughts on your other comments and I'll update this PR.